e2e/utils.go

```diff
@@ -279,6 +279,7 @@ func createCephfsStorageClass(c kubernetes.Interface, f *framework.Framework, en
 	if enablePool {
 		sc.Parameters["pool"] = "myfs-data0"
 	}
+
```

Comment:
I like the additional extra line here, but it should not be part of this PR

Reply:
removed.